tools/webpack/packages.js

```diff
@@ -23,11 +23,7 @@ const WORDPRESS_NAMESPACE = '@wordpress/';
 // Experimental or other packages that should be private are bundled when used.
 // That way, we can iterate on these package without making them part of the public API.
 // See: https://github.com/WordPress/gutenberg/pull/19809
-const BUNDLED_PACKAGES = [
-	'@wordpress/icons',
-	'@wordpress/interface',
-	'@wordpress/style-engine',
-];
+const BUNDLED_PACKAGES = [ '@wordpress/icons', '@wordpress/interface' ];
```

Comment:
@youknowriad Do you remember? Is this the only change required to ensure that the JS methods appear in the global window scope? E.g., `wp.styleEngine`

I couldn't seem to get it working.

Reply:
It needs to get removed in all places where `@wordpress/style-engine` is explicitly marked as an exception like here:
 
https://github.com/WordPress/gutenberg/blob/ce4feda1038708b563b7f98ae6b886671dc56945/packages/dependency-extraction-webpack-plugin/lib/util.js#L2-L6

Well, it's probably the only place left 😄